libfreerdp/core/gateway/rdg.c

```diff
@@ -323,34 +359,313 @@ static BOOL rdg_write_packet(rdpRdg* rdg, wStream* sPacket)
 	return TRUE;
 }
 
+static BOOL rdg_write_websocket(rdpRdg* rdg, wStream* sPacket)
+{
+	size_t s;
+	int status;
+	wStream* sWS;
+
+	uint32_t maskingKey;
+
+	int streamPos;
+
+	s = Stream_Length(sPacket);
```

Comment:
would be nice to have a variable with another name, we usually use `s` for `stream`, not for `size` (len ?).